.config/yadm/bootstrap.d/bootstrap.100-install-packages.sh##o.Linux,e.sh

```diff
@@ -0,0 +1,179 @@
+    fi
+    
+    # Install lazydocker
+    if ! command -v lazydocker &>/dev/null; then
+        echo "Installing lazydocker..."
+        local lazydocker_version=$(curl -s https://api.github.com/repos/jesseduffield/lazydocker/releases/latest | grep '"tag_name"' | cut -d'"' -f4)
+        curl -L "https://github.com/jesseduffield/lazydocker/releases/download/${lazydocker_version}/lazydocker_${lazydocker_version#v}_Linux_x86_64.tar.gz" -o /tmp/lazydocker.tar.gz
+        tar -xzf /tmp/lazydocker.tar.gz -C /tmp/
+        mv /tmp/lazydocker "$bin_dir/"
+        chmod +x "$bin_dir/lazydocker"
+        rm /tmp/lazydocker.tar.gz
+    fi
+    
+    # Install lazygit
+    if ! command -v lazygit &>/dev/null; then
+        echo "Installing lazygit..."
+        local lazygit_version=$(curl -s https://api.github.com/repos/jesseduffield/lazygit/releases/latest | grep '"tag_name"' | cut -d'"' -f4)
+        curl -L "https://github.com/jesseduffield/lazygit/releases/download/${lazygit_version}/lazygit_${lazygit_version#v}_Linux_x86_64.tar.gz" -o /tmp/lazygit.tar.gz
+        tar -xzf /tmp/lazygit.tar.gz -C /tmp/
+        mv /tmp/lazygit "$bin_dir/"
+        chmod +x "$bin_dir/lazygit"
+        rm /tmp/lazygit.tar.gz
+    fi
+    
+    # Install zoxide
+    if ! command -v zoxide &>/dev/null; then
+        echo "Installing zoxide..."
+        curl -sSfL https://raw.githubusercontent.com/ajeetdsouza/zoxide/main/install.sh | sh
+    fi
+    
+    # Install zellij
+    if ! command -v zellij &>/dev/null; then
+        echo "Installing zellij..."
+        local zellij_version=$(curl -s https://api.github.com/repos/zellij-org/zellij/releases/latest | grep '"tag_name"' | cut -d'"' -f4)
+        curl -L "https://github.com/zellij-org/zellij/releases/download/${zellij_version}/zellij-x86_64-unknown-linux-musl.tar.gz" -o /tmp/zellij.tar.gz
+        tar -xzf /tmp/zellij.tar.gz -C /tmp/
+        mv /tmp/zellij "$bin_dir/"
+        chmod +x "$bin_dir/zellij"
+        rm /tmp/zellij.tar.gz
+    fi
+    
+    # Install uv
+    if ! command -v uv &>/dev/null; then
+        echo "Installing uv..."
+        curl -LsSf https://astral.sh/uv/install.sh | sh
+    fi
+    
+    # Install Bob (Neovim version manager)
+    if ! command -v bob &>/dev/null; then
+        echo "Installing Bob (Neovim version manager)..."
+        curl -sSf https://raw.githubusercontent.com/MordechaiHadad/bob/master/install | bash -s -- --to "$bin_dir"
+    fi
+    
+    # Install aichat
+    if ! command -v aichat &>/dev/null; then
+        echo "Installing aichat..."
+        local aichat_version=$(curl -s https://api.github.com/repos/sigoden/aichat/releases/latest | grep '"tag_name"' | cut -d'"' -f4)
+        curl -L "https://github.com/sigoden/aichat/releases/download/${aichat_version}/aichat-${aichat_version}-x86_64-unknown-linux-musl.tar.gz" -o /tmp/aichat.tar.gz
+        tar -xzf /tmp/aichat.tar.gz -C /tmp/
+        mv "/tmp/aichat-${aichat_version}-x86_64-unknown-linux-musl/aichat" "$bin_dir/"
```

Comment:
please double check the installation of aichat.
When I tried it, I got "directory not found" errors